src/test/java/core/basesyntax/service/impl/CsvFileWriterImplTest.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.CsvFileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import org.junit.BeforeClass;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class CsvFileWriterImplTest {
+    private static final String OUTPUT_FILE_PATH = "src/test/java/resources/reportTest.cvs";
+    private static final String OUTPUT_FOLDER_PATH = "src/test/java/resources/";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private static final String REPORT = "fruit,quantity" + LINE_SEPARATOR
+            + "banana,10" + LINE_SEPARATOR
+            + "apple,10";
+    private static CsvFileWriter writer;
+    @Rule
+    public final ExpectedException exception = ExpectedException.none();
+
+    @BeforeClass
+    public static void setUp() {
+        writer = new CsvFileWriterImpl(OUTPUT_FILE_PATH);
+    }
+
+    @Test
+    public void write_correctReport_ok() {
+        writer.write(REPORT);
+        try {
+            String report = String.join(LINE_SEPARATOR,
+                    Files.readAllLines(Path.of(OUTPUT_FILE_PATH)));
+            assertEquals("Test failed! Expected report: " + REPORT + LINE_SEPARATOR
+                    + "But actual is: " + report, REPORT, report);
+        } catch (IOException e) {
+            throw new RuntimeException("File not found '" + OUTPUT_FILE_PATH + '\'');
+        }
```

Comment:
test isn't the best place to have try-catch, here just add `throws IOException` to the method signature